Automattic/wp-calypso
# Posts: Update hover information on author selector

PR to address some tooltip weirdness in #6010. Currently, when you hover over the author navigation selectors on the Posts page in Calypso, you'll see hover text like `Me,[object, Object]`. This PR specifies a title attribute for the author navigation selectors so that you'll only see the desired text.

With the title attribute omitted, the nav item title field automatically picks up the inner text, which includes the desired text but also some info that determines whether or not to display the Gravatar like this:

```
{ textItem }
{ isMe && <Gravatar size={ 16 } user={ user.get() } /> }
```

We just need the `textItem` attribute in this case.

## To test
1. Load up this branch and visit the Posts page for a multi-author site on your account: https://wordpress.com/posts/site.wordpress.com
2. Hover over the "Me" and "Everyone" selectors at the top of the page. On hover, you should only see "Me" or "Everyone." If you visit this same page in WordPress.com, you'll have extra text added on to the hover tooltip.

## Screenshot

![tooltip](https://cloud.githubusercontent.com/assets/7240478/16151680/9849bf50-345b-11e6-9eba-fcd1ae6ae80f.png)


Test live: https://calypso.live/?branch=fix/6010-posts-tooltip
